concent_api/core/views.py

```diff
@@ -79,6 +79,43 @@ def receive_out_of_band(_request):
     return HttpResponse("out of band message sent", status = 200)
 
 
+def validate_message(data):
+    if 'type' not in data:
+        raise Http400("Unspecified message data type.")
```

Comment:
You should return 4xx and 5xx error messages in JSON format. Not as a plain string.